lib/utils/tracking.js

```diff
@@ -55,7 +51,7 @@ function request(type, event, { id, timeout } = {}) {
     },
     method: 'POST',
     // set to 1000 b/c no response needed
```

Comment:
Super nit, but not super important to update

Reply:
Updated with something more accurate